hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc

```diff
@@ -24,7 +24,7 @@ include::modules/hcp-virt-reqs.adoc[leveloffset=+1]
 
 * xref:../../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#recommended-etcd-practices[Recommended etcd practices]
 * xref:../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc[Persistent storage using LVM Storage]
-* To disable the {hcp} feature or, if you already disabled it and want to manually enable it, see _Enabling or disabling the {hcp} feature_.
+* To disable the {hcp} feature or, if you already disabled it and want to manually enable it, see xref:../../hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc[Enabling or disabling the {hcp} feature].
```

Comment:
Nit - I think even if you're linking to the overall page, you should still use the ID of the assembly for the anchor here: 
```suggestion
* To disable the {hcp} feature or, if you already disabled it and want to manually enable it, see xref:../../hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc#hcp-enable-disable[Enabling or disabling the {hcp} feature].
```